packages/theme/brand/altinn.css

```diff
@@ -1331,4 +1331,196 @@
   --ds-letter-spacing-8: 0.5%;
   --ds-letter-spacing-9: 1.5%;
 }
-}
\ No newline at end of file
+}
+/**
+ * These files are generated from design tokens defind using Token Studio
+ */
+
+@layer ds.theme.color {
+[data-color="accent"] {
```

Comment:
Should we maybe assign the first "color" as the `:root` color? We could enforce the need for a "color" and `neutral` as bare minimum for our color. 

The current setup will with `neutral` defined on `:root` will make the default color B&W.

Reply:
After implementing color inheritance in another branch, I agree with your assessment – changing the default `:root` color to `accent`.